neptune/new/exceptions.py

```diff
@@ -770,18 +800,41 @@ def __init__(self):
 class NeptuneOfflineModeChangeStageException(NeptuneOfflineModeException):
     def __init__(self):
         message = """
-        TODO: NPT-11349
+{h1}
+----NeptuneOfflineModeChangeStageException---------------------------------------
+{end}
+You cannot change the stage of the model version while in OFFLINE mode.
 """
         super().__init__(message.format(**STYLES))
 
 
 class NeptuneProtectedPathException(NeptuneException):
+    extra_info = ""
+
     def __init__(self, path: str):
         message = """
-        TODO: NPT-11349
+{h1}
+----NeptuneProtectedPathException----------------------------------------------
+{end}
+Field {path} cannot be changed directly.
+{extra_info}
+
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help
 """
         self._path = path
-        super().__init__(message.format(**STYLES))
+        super().__init__(
+            message.format(
+                path=path,
+                extra_info=self.extra_info.format(**STYLES),
+                **STYLES,
+            )
+        )
+
+
+class NeptuneCannotChangeStageManually(NeptuneProtectedPathException):
+    extra_info = """
+If you want to change the stage of the model version,
+use the {python}.change_stage(){end} function."""
```

Comment:
![image](https://user-images.githubusercontent.com/14312755/153872342-b069aa23-1ede-4a22-ad2d-8a498ef8e862.png)

Reply:
I would provide a full example:
`If you want to change the stage of the model version,
use the {python}.change_stage(){end} function:
    {python}model_version.change_stage("staging"){end}`

Reply:
update, ![image](https://user-images.githubusercontent.com/14312755/153889174-ac0f65f9-ada5-4a1b-8942-cb97066d79a3.png)